tools/Makefile

```diff
@@ -128,6 +129,7 @@ $(curdir)/quilt/compile := $(curdir)/autoconf/compile $(curdir)/findutils/compil
 $(curdir)/sdcc/compile := $(curdir)/bison/compile
 $(curdir)/squashfs3-lzma/compile := $(curdir)/lzma-old/compile
 $(curdir)/squashfs4/compile := $(curdir)/xz/compile $(curdir)/zlib/compile
+$(curdir)/util-linux/compile := $(curdir)/meson/compile $(curdir)/bison/compile
```

Comment:
As you are using the autotools build system the dependency on meson should be unnecessary.

Reply:
Yep. I was trying meson first but it can't precisely select the minimum set of programs to build. That's why now autotools build system is used.
I'll remove this line.